src/runtime/components/Table.vue

```diff
@@ -247,5 +279,10 @@ defineExpose({
         </tr>
       </tbody>
     </table>
+    <slot name="pagination">
+      <div :class="ui.pagination({ class: [props.ui?.pagination] })">
+        <UPagination v-if="tableApi.getPageCount() > 1" v-model:page="page" :items-per-page="1" :total="tableApi.getPageCount()" />
+      </div>
+    </slot>
```

Comment:
Not sure about this, I would not include the `UPagination` component inside the Table to let users handle it the way they want and put it wherever they want I guess.